Server/src/services/authenticationService.ts

```diff
@@ -174,14 +174,14 @@ export class AuthenticationService {
    * each request will contain an access token and its signature will be verified
    * @param jwtParams Parameters used to build JWT Token
    */
-  private async generateJwtToken(jwtPayload: IJwtParams): Promise<string> {
+  private generateJwtToken(jwtPayload: IJwtParams): string {
 
     let token: string;
     let tokenInformation;
     let jwtExpiry: number = 3000;
     const jwtAccessKey = process.env.ACCESS_SECRET_KEY;
 
-    token = await jwt.sign({ jwtPayload }, jwtAccessKey, {
+    token = jwt.sign({ jwtPayload }, jwtAccessKey, {
```

Comment:
jwt.sign returns a promise, this call should be awaited, why did you remove await?

Reply:
sign method is actually synchronous apparently, sonarqube flagged it and checked the code
here's part of the description from source code
"Synchronously sign the given payload into a JSON Web Token string payload "

Reply:
can you attach a screenshot? its pretty weird, im surprised how sonarCube knows something extra we cant tell ourselves

Reply:
wait, weird intellisense told me it returns a promise before, now it says string, resolved 👍

Reply:
sorry I don't have the screenshot from sonarqube as i already marked the code smell as fixed, but the sign we are using in the code is the synchronous one without callback 
![image](https://user-images.githubusercontent.com/6520150/109901918-d01d7800-7c67-11eb-9458-e1a7d79d2b5d.png)

Reply:
![image](https://user-images.githubusercontent.com/6520150/109902172-0d820580-7c68-11eb-9082-51322c75aa12.png)

Reply:
> wait, weird intellisense told me it returns a promise before, now it says string, resolved 👍

sounds good, thank you for double checking!